### ISSUE TITLE
PortalGun music discs do not play

### ISSUE THREAD

**Issue by Chrriiisss** [xJon/The-1.7.10-Pack#1949]
Fresh installation of 1.7.10 modpack and server.

Crafted the music disc via creating a weighted cube -> companion -> music disc. Crafted into a radio and placed down, no music can be heard. Spawned in the music disc again and place it into a jukebox, no sound.

Tested with all other PortalGun mod music discs and all discs do not produce music/sound. Testing with vanilla music discs works perfectly.

I have checked that my sound settings were correct and brought another player on the server to check it wasn't just me, still no sound heard.

In Summary: Both the Radio, portal radio music disc and still alive music disc are non-functional.

**Comment by xJon** [comment]
Seems to also be reported over https://github.com/iChun/IssueTracker-Mods/issues/183, and [CurseForge](https://www.curseforge.com/minecraft/mc-mods/portal-gun?filter-game-version=1738749986:5&page=2&sort=-game-version#comments).
